packages/cli/src/commands/governance/votePartially.ts

```diff
@@ -13,11 +13,9 @@ export default class VotePartially extends BaseCommand {
   static flags = {
     ...BaseCommand.flags,
     proposalID: flags.string({ required: true, description: 'UUID of proposal to vote on' }),
-    values: flags.string({
-      required: true,
-      description: "Array of votes split by ','",
-    }),
-    weights: flags.string({ required: true, description: "Array of weights split by ','" }),
+    yes: flags.string({ required: true, description: 'Yes votes' }),
```

Comment:
Maybe you could remove the required validation here, and assume zero if it was not present